### ISSUE TITLE
Low Processor Mode -> missing screen update on NotificationWmFocusIn 

### ISSUE THREAD

**Issue by Difference** [godotengine/godot#52008]
### Godot version

3.3.2

### System information

IOS

### Issue description

When turning on Low Processor Mode, returning to app, does not refresh the screen, resulting in a black screen, until you interact with the app.

Users will likely assume the app has crashed.

A workaround is to play a small animation on MainLoop.NotificationWmFocusIn like so ( ResumeLabel has a tween that displays it, then fades out)

```
public override void _Notification(int what)
    {
        base._Notification(what);
           if (what == MainLoop.NotificationWmFocusIn  )
            {
              var reslab = (ResumeLabel)FindNode("ResumeLabel");
               reslab.Play();
            }

            if (what == MainLoop.NotificationWmFocusOut )
            {
            var reslab = (ResumeLabel)FindNode("ResumeLabel");
            reslab.Visible = false;
            }    
    }

```

### Steps to reproduce

Turn on Low Processor Mode , run on IOS device, and let device goto sleep. ( or maybe just not have focus) until low processor mode kicks in.
Return to app, to see a blank/black screen.
Interact with app, to see screen turn on.

**Comment by Calinou** [comment]
Low processor mode has many bugs on mobile in general. Is it usable on iOS outside of this bug?

**Comment by Zireael07** [comment]
Possibly related to 51946?

**Comment by Difference** [comment]
> Low processor mode has many bugs on mobile in general. Is it usable on iOS outside of this bug?

Yes, I benchmarked, and it reduces CPU usage when it kicks in. I do not have any other problems with it in my In production App ( https://apps.apple.com/dk/app/krydsordmen/id1393999562 )

**Comment by maran** [comment]
I'm also experiencing this on 3.5.1, it even happens on first boot for me so the app boots into a black screen giving users the feeling the app crashed.
